static/js/loading.js

```diff
@@ -44,9 +46,6 @@ exports.make_indicator = function (outer_container, opts) {
     // but that's probably OK for our purposes.
     container.css({width: 38 + text_width,
                    height: 38});
-    outer_container.css({display: 'block',
-                         'white-space': 'nowrap'});
-
```

Comment:
If you're going to move this comment, you should move the `TODO` comment that goes with it (just above).